algolia.js

```diff
@@ -105,49 +99,58 @@ async function index() {
       item.locales = [item.locale];
     }
     item.indexedOn = indexedOn.getTime();
-    return item;
+    return trimBytes(item);
   });
 
-  const chunkedAlgoliaData = chunkAlgolia(algoliaData);
-  const postsCount = algoliaData.length;
-
-  // @ts-ignore
-  const client = algoliasearch(
-    process.env.ALGOLIA_APP_ID,
-    process.env.ALGOLIA_API_KEY,
-  );
-  const index = client.initIndex('prod_web_dev');
+  // This exports the parsed data for a test
+  if (process.env.ELEVENTY_ENV === 'test') {
```

Comment:
This means that this script does entirely different thing on test and on real use. This renders testing completely obsolete.